automation/packaging/rpm/el9/checksum.sh

```
@@ -0,0 +1,31 @@
+fi
+
+pushd rpmbuild/RPMS/x86_64
+trap 'popd' EXIT
+
+echo "VERSION: $VERSION"
+echo "RELEASE: $RELEASE"
+
+RPM_FILE="flex-${VERSION}-${RELEASE}.x86_64.rpm"
+if [ ! -f "$RPM_FILE" ]; then
+        echo "ERROR: Expected RPM file $RPM_FILE not found"
+        ls -la *.rpm 2>/dev/null || echo "No RPM files in directory"
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)